### ISSUE TITLE
Swift Package Manager + Xcode 12.0:  Target 'arm64-apple-ios-simulator' missing

### ISSUE THREAD

**Issue by jzeisweiss** [apollographql/apollo-ios#1403]
Hello,

I can no longer use "Preview" for SwiftUI or build to a simulator due to the following error.:

```
error: could not find module 'Apollo' for target 'arm64-apple-ios-simulator'; found: x86_64-apple-ios-simulator, x86_64
import Apollo
```
**Xcode Version**: 12.0 (12A7209)
**Apollo Version**: 33.0

_Note: I am having no problem building to a device._

**Comment by designatednerd** [comment]
@jzeisweiss Hi, what package manager are you using?

**Comment by jzeisweiss** [comment]
@designatednerd - Hey Ellen, I am using Swift Package Manager.

**Comment by designatednerd** [comment]
Are you using it on the Developer Transition Kit (I ask since it's looking for the `arm64-apple-ios-simulator` slice)? They took all the Apple Silicon stuff out of 12.0 and put it back in the 12.2 beta, so you'd need to use that if you are.

**Comment by jzeisweiss** [comment]
I am not using the Silicon ARM-Based prototype Mac from the Developer Transition Kit. I am using a 2018 Mac Mini and a 2018 MacBook Pro. However, I did find a workaround. I went in and changed the project settings to:
"Build Settings" > "Build Active Architecture Only" > Set to "YES". This fixed my issue and I can now see the SwiftUI previews as well as build to simulators. Thank you for the speedy responses!

**Source**: https://stackoverflow.com/questions/56957632/could-not-find-module-for-target-x86-64-apple-ios-simulator
